src/content/docs/cheerp/guides/clientlib.md

```diff
@@ -0,0 +1,327 @@
+---
```

Comment:
I think `guides` is the wrong place for this, uh, guide, at least the root of `guides` is anyway. We should probably include it as part of an upgrade guide section ([like Next.js's](https://nextjs.org/docs/app/building-your-application/upgrading)). CheerpJ sort of has this on the [Migration fro CheerpJ 2](https://labs.leaningtech.com/cheerpj3/migrating-from-cheerpj2) page. What do you think?

On a similar note, when is new clientlib shipping? Is it available on nightly today? If so, we should say so. We should be thinking about a path to stability (i.e. Cheerp 3.1) sooner rather than later too IMO although I know that's contraversial.

I'm kinda brainstorming a `version` property that you can specify in page frontmatter that defines a semver range for versions where a certain feature is supported. Then we can show a banner near the top of the page explaining the situation, or do something more clever in future.

Reply:
I've moved it out of the guides section for now. I don't like having an upgrade section with only one page.

New clientlib will be on nightly Soon™.